transports/webrtc-utils/README.md

```diff
@@ -0,0 +1,6 @@
+# WebRTC Common Utilities
+
+Tools that are common to more than one WebRTC component.
+
+-   [Protobuf Generated Code](generated)
+-   [Substream State](substream/state.rs)
```

Comment:
Sounds like a good idea, for after everything settles and we're confident that all the common items are identified?

Reply:
Yes, it might be best to leave this near the end once I'm more confident we got all the common portions, to minimize re-working it each time

Reply:
So I think this might add too much complexity, as the `PollDataChannel` (the Data Channel wrapper for async read/write) implements `AsyncRead + AsyncWrite` from two different libraries: one is `tokio` and the other is `futures`. It might add more complexity than it is worth to get these into a common crate